WordPress/Classes/ViewRelated/NUX/SignupUsernameTableViewController.swift

```diff
@@ -216,13 +216,14 @@ extension SignupUsernameTableViewController {
         switch indexPath.section {
         case Sections.suggestions.rawValue:
             if indexPath.row == 0 {
-                selectedUsername = ""
+                selectedUsername = currentUsername ?? ""
```

Comment:
`""` is used on line `130` of `SignupEpilogueViewController.swift` to decide if the username change request needs to be performed. With this change, tapping a username and then tapping the original will cause a username change request—it will succeed, but it will hit the network and show the user the "changing username" HUD.

Reply:
Thanks @nheagy . I'll address that in another PR.